.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to commit `.project?`

Reply:
Not sure... I think it got pulled in from chris' changes to cmake unfortunately. @Chrisc110?

Reply:
@sahil-kale no, we do not need to commit .project, but could it be helpful for ... making sure that when you open a project in cubeide all the files/folders are there already?

Reply:
the reason you typically avoid commiting IDE files is to avoid the diff showing up unnecessarily - every PR will contain this diff and it's bloat that doesn't need to get pulled in or committed. This will cause unnecessary merge conflicts and cause editor setups to vary every time code is pulled. 

We should not commit this file unless the project does not work without it.